src/log.cpp

```diff
@@ -207,6 +208,25 @@ static void print_precise_timestamp(std::ostream& out) noexcept
 	} catch(...) {}
 }
 
+std::string sanitize_log(const std::string& path)
```

Comment:
It'd be good to call this something other than `path` and `tmp_path`, since these aren't file paths anymore.